widgetastic_manageiq.py

```diff
@@ -425,3 +431,126 @@ class Accordion(PFAccordion):
     def is_dimmed(self):
         return bool(
             self.browser.elements('.//div[contains(@id, "tree") and contains(@class, "dimmed")]'))
+
+
+class Paginator(Widget):
+    """ Represents Paginator control that includes First/Last/Next/Prev buttons
+    and a control displaying amount of items on current page vs overall amount.
+
+    It is mainly used in Paginator Pane.
+    """
+    ROOT = '//div[@id="pc_div_1"]/*/ul[@class="pagination"]/'
```

Comment:
@mfalesni I thought about parent's id but I decided to update widget when I'd see Paginator control somewhere else except pagination pane.

Anyway, I fixed this. Now paginator retrieves root path from parent's control and joins it with its relative path.